tests/sentry/web/frontend/test_2fa.py

```diff
@@ -3,13 +3,14 @@
 from django.core.urlresolvers import reverse
 
 from sentry.testutils import TestCase
-from sentry.models import TotpInterface
+from sentry.models import TotpInterface, User
 
 
-class TwoFactorAuthTest(TestCase):
+class TwoFactorAuthTestPassword(TestCase):
 
     def test_security_renders_without_2fa(self):
-        user = self.create_user('foo@example.com')
+        User.objects.create(email='a@example.com')
```

Comment:
Not really, no. Is the create_user method better to use in this instance or are they functionally the same? I'm trying to test signing in as a user and accessing the security settings page in different instances of having 2fa enabled/disabled.